docs/topics/native/native-spm.md

```diff
@@ -6,47 +6,48 @@ Consider a Kotlin Multiplatform project that has an iOS target. You may want to
 as a dependency to iOS developers working on native Swift projects. Using Kotlin Multiplatform tooling, you can provide
 an artifact that would seamlessly integrate with their Xcode projects.
 
-This guide shows how to do this by building [XCFrameworks](multiplatform-build-native-binaries.md#build-xcframeworks)
+This tutorial shows how to do this by building [XCFrameworks](multiplatform-build-native-binaries.md#build-xcframeworks)
 with the Kotlin Gradle plugin.
 
-## Prepare file locations
+## Set up remote integration
 
-To make your framework consumable, you need to upload two files:
-* A ZIP archive of the XCFramework. Upload it to a convenient file storage with direct access (for example,
-creating a GitHub release with the archive attached, using Amazon S3 or Maven).
+To make your framework consumable, you'll need to upload two files:
+
+* A ZIP archive with the XCFramework. You'll need to upload it to a convenient file storage with direct access (for example,
+  creating a GitHub release with the archive attached, using Amazon S3 or Maven).
   Choose the option that is easiest to integrate into your workflow.
-* The `Package.swift` file describing the package. Prepare a Git repository and push it there.
-  To decide how to organize your repositories, see the pros and cons of several options below.
+* The `Package.swift` file describing the package. You'll need to push it to a separate Git repository.
 
-### Swift package distribution
+#### Project configuration options {initial-collapse-state="collapsed"}
 
-Consider the following options for organizing your Git repositories:
+In this tutorial, you'll store your XCFramework and the `Package.swift` file in separate Git repositories.
+However, there can be other project configurations. You can consider the following options for organizing your Git repositories:
 
-* Store the `Package.swift` file in an independent repository. This allows versioning it separately from the
-Kotlin Multiplatform project the file describes. This is the recommended approach: it allows scaling and generally easier
-to maintain.
+* Store the `Package.swift` file and your ZIP archive with the XCFramework in separate Git repositories. This allows
+  versioning it separately from the Kotlin Multiplatform project the file describes. This is the recommended approach:
+  it allows scaling and is generally easier to maintain.
```

Comment:
I agree with this. targets can be defined as follows:

```swift
targets: [
    .binaryTarget(
        name: "shared",
        path: "./shared.xcframework"
    )
]
```

binaries can be versioned and avoids need for zipping, generating checksums and uploading to remote hosted store.

Reply:
According to our latest survey and further interviews, teams use SPM integration mainly for remote integration. This is because of a request from the iOS team: we use SPM in our iOS app, so you need to find a way to integrate KMP in the same way. 

But I believe we can add this as an alternative scenario. Thanks for your suggestion!